src/constructor.js

```diff
@@ -45,6 +45,12 @@ class TodoList {
     this.list.find((item) => item.index === parseInt(index, 10)).completed = value;
     this.save();
   }
+
+  deleteCompleted() {
+    this.list.filter((item) => item.completed).forEach((element) => {
+      this.delete(element.index);
+    });
+  }
```

Comment:
- [ ] In the project's specific requirements a function for editing task descriptions needs to be implemented. You might implement it in this TodoList class or anywhere that you think is convenient. Kindly implement it. If you want to take a look back at the project's requirements here is the [link](https://github.com/microverseinc/curriculum-javascript/blob/main/todo-list/m3_add_remove.md). :+1: